gift-certificates-app/dao/src/main/java/com/epam/esm/bean/Certificate.java

```diff
@@ -111,36 +73,24 @@ public boolean equals(Object o) {
         Certificate that = (Certificate) o;
 
         if (id != that.id) return false;
-        if (Double.compare(that.price, price) != 0) return false;
-        if (duration != that.duration) return false;
         if (!Objects.equals(name, that.name)) return false;
         if (!Objects.equals(description, that.description)) return false;
+        if (!Objects.equals(price, that.price)) return false;
+        if (!Objects.equals(duration, that.duration)) return false;
         if (!Objects.equals(createDate, that.createDate)) return false;
-        if (!Objects.equals(lastUpdateDate, that.lastUpdateDate))
-            return false;
-        return Objects.equals(tagList, that.tagList);
+        return Objects.equals(lastUpdateDate, that.lastUpdateDate);
     }
 
     @Override
     public int hashCode() {
```

Comment:
you can use @Data from lombock library to avoid all the boilerplate

Reply:
+